.gitignore

```diff
@@ -240,3 +243,6 @@ lerna-debug.log
 
 # mocha
 **/test-results.*.xml
+
+# release scripts
+.scripts/
```

Comment:
I don't think this folder should be ignored by git - after all, you're checking stuff in right now that goes into it.

Reply:
I added a more specific folder name to exclude the node_modules dependencies folder created by npm install in order to run bump-version.

---
In reply to: [265818667](https://github.com/Azure/azure-iot-sdk-node/pull/510#discussion_r265818667) [](ancestors = 265818667)